recipe/windows-bat.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/graphviz/backend.py b/graphviz/backend.py
+index 955551a..be35745 100644
+--- a/graphviz/backend.py
++++ b/graphviz/backend.py
+@@ -69,6 +69,8 @@ def command(engine, format, filepath=None):
+     if format not in FORMATS:
+         raise ValueError('unknown format: %r' % format)
+ 
++    if PLATFORM == 'windows':
++        engine = '%s.bat' % engine
```

Comment:
I can confirm that it does work with this patch (AppVeyor fails from `'curl' is not recognized as an internal or external command`). Note that `shell=True` might be regarded problematic for security reasons (potential shell injection with a user-supplied filename string).

IMHO, a  better solution would be to not have the `.bat` wrappers from https://github.com/conda-forge/graphviz-feedstock/ in the first place, if that is possible.